lib/screens/stops/stop.ex

```diff
@@ -255,6 +258,9 @@ defmodule Screens.Stops.Stop do
            [],
            true
          ) do
+      {:ok, %{"data" => []}, _} ->
+        :bad_response
```

Comment:
The function that uses this function looks for :error atom, not :bad_response, so it will be unable to match. Also, would be great to have the error logged in `create_station_with_routes_map`. And finally, we should probably let the empty list fail more gracefully down the road -- so if we log the error in `create_station_with_routes_map` but still pass an empty list, we should handle that empty list better in departures.ex.